enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk/rspack.embed.config.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable import/no-commonjs */
+/* eslint-disable no-undef */
+
+const path = require("path");
+
+const SCRIPT_TAG_PATH = path.resolve(__dirname, "embed.v1.ts");
+
+const BUILD_PATH = path.resolve(
+  __dirname,
+  "../../../../../resources/frontend_client",
+);
+
+module.exports = {
+  name: "iframe_sdk_embed_v1",
+  entry: SCRIPT_TAG_PATH,
+  output: {
+    path: BUILD_PATH + "/app",
+    filename: "embed.v1.js",
+    library: "metabase.embed",
+    libraryTarget: "umd",
+    globalObject: "this",
+  },
+  devServer: { hot: false },
+  watchOptions: {
+    ignored: ["**/node_modules", SCRIPT_TAG_PATH],
+  },
```

Comment:
HMR is having issues on the script tag where it's generating `hot-update` files at `main` (as we are using `devServer` on the main rspack config) without ever cleaning them up, so we have to ignore the SCRIPT_TAG_PATH. I think it's a rspack bug that happens when we re-export multiple rspack configs and they have differing `devServer` setups